xbmc/Util.cpp

```diff
@@ -2311,7 +2311,15 @@ void CUtil::ScanForExternalAudio(const std::string& videoPath, std::vector<std::
   GetItemsToScan(strBasePath, CServiceBroker::GetFileExtensionProvider().GetMusicExtensions(), common_sub_dirs, items);
 
   std::vector<std::string> exts = StringUtils::Split(CServiceBroker::GetFileExtensionProvider().GetMusicExtensions(), "|");
-  ScanPathsForAssociatedItems(strAudio, items, exts, vecAudio);
+
+  const bool skipNamesCheck = CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_VIDEOPLAYER_DONTCHECKEXTERNALAUDIONAMES);
+  if (skipNamesCheck) {
+      for (const auto& audioItem : items.GetList()) {
```

Comment:
indentation is 2 spaces, move curly bracket into a separate line

Reply:
Fixed.